java/Makefile

```diff
@@ -31,3 +31,5 @@ android-shell:
 logcat:
 		${ADB} logcat
 
+uninstall:
+		${ADB} uninstall ${PACKAGE}
```

Comment:
do we have a mvn uninstall as well? might be lovely for cleaning up bits installed with -P deploy